podpac/core/data/interpolate.py

```diff
@@ -1093,15 +1096,19 @@ def _select_interpolator_queue(self, source_coordinates, eval_coordinates, selec
                 # see which dims the interpolator can handle
                 can_handle = getattr(interpolator, select_method)(udims, source_coordinates, eval_coordinates)
 
-                # if interpolator can handle all udims
-                if not set(udims) - set(can_handle):
-
+                # dims handled by this interpolator
+                this_handled_dims = set(can_handle) - set(handled_dims)
+                
+                # if interpolator can handle some udims
+                if this_handled_dims:
```

Comment:
Again, this block of code allows multiple interpolators to be selected as part of the queue. 

I just realized that the original code allowed a single interpolator to handle all the dimensions if possible. A little bit of logic should allow that again. 

I.e. A can handle time, B can handle lat-lon, C can handle time lat-lon and alt, so just use C for all. (instead of A for time, B for lat-lon, C for alt based on the current implementation.)